EncoreTickets.SDK/BaseApi.cs

```diff
@@ -118,11 +119,11 @@ public Stream GetResource(string url)
             else
             {
                 rr = client.Execute(request);
-                T rawData = SimpleJson.SimpleJson.DeserializeObject<T>(rr.Content);
+                var rawData = SimpleJson.SimpleJson.DeserializeObject<T>(rr.Content);
                 apiResponse = new ApiResponse<T>(rawData);
             }
 
-            return new ApiResult<T>(this.context, request, rr, apiResponse);
+            return new ApiResult<T>(context, request, rr, apiResponse);
```

Comment:
While I dislike using ```this```, I think that if we are using the same naming convention for both private fields and local variables, we should be prefixing fields with ```this``` to avoid confusion. Alternatively, we can try changing the naming convention so that, for example, field names are prefixed with ```_```